cfgov/v1/preprocessed/css/molecules/image-and-text-25-75.less

```diff
@@ -0,0 +1,93 @@
+/* topdoc
+  name: Image and Text 25/75
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-image-and-text-25-75">
+            <div class="m-image-and-text-25-75_image-container">
+                <img src="image">
+            </div>
+            <div class="m-image-and-text-25-75_content">
+                <h1 class="m-image-and-text-25-75_heading">Heading</h1>
+                <p class="m-image-and-text-25-75_body">Body content</p>
+                <a class="m-image-and-text-25-75_link">Link</a>
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-image-and-text-25-75
+            .m-image-and-text-25-75_image-container
+              img
+            .m-image-and-text-25-75_content
+              .m-image-and-text-25-75_heading
+              p.m-image-and-text-25-75_body
+              a.m-image-and-text-25-75_link
+  tags:
+    - cfgov-molecules
+*/
+
+.m-image-and-text-25-75 {
+
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+
+    &_image-container {
+
+        > img {
+            width: 130px;
+
+            .respond-to-min( @desktop-min, {
+                width: 150px;
+            } );
+        }
+        text-align: center;
+        margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
+
+        .respond-to-min( @tablet-min, {
+            text-align: left;
+            float: left;
+            margin-right: unit( @grid_gutter-width / @base-font-size-px, em );
+            margin-bottom: 0;
+        } );
+    }
+
+    &_content {
+        .respond-to-min( @tablet-min, {
+            margin-left: unit( ( 130px + @grid_gutter-width ) / @base-font-size-px, em );
+        } );
+
+        .respond-to-min( @desktop-min, {
+            margin-left: unit( ( 150px + @grid_gutter-width ) / @base-font-size-px, em );
+        } );
+    }
+
+    &_heading,
+    &_body,
+    &_link {
+        /* TODO: Margin is wider than expecting when applied to h2.
+        margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em ); */
+        margin-bottom: 30px;
+    }
+
+    &_link {
+        .jump-link();
+    }
+
+    &_content::after {
+        content: '';
+        clear: both;
+    }
+}
+
+h1.m-image-and-text-25-75_heading {
+    /* TODO: Margin is wider than expecting when applied to h1.
+    margin-bottom: unit( ( @grid_gutter-width * 2 ) / @base-font-size-px, em ); */
+    margin-bottom: 60px;
```

Comment:
That's because you're using `@base-font-size` but the `h1` is 34px at > 600 screens and 26px < 600 screens. Same for above. There's actually a hack around this that I had to put a fix in for in cf-core because I broke it. By calling `.h1();` we expose a `@font-size` var that you could then use in place of `@base-font-size` and it adjusts itself automatically at the 600px bp.